pkg/kudoctl/cmd/init.go

```diff
@@ -155,3 +174,40 @@ func (i *initCmd) YAMLWriter(w io.Writer, manifests []string) error {
 	_, err := fmt.Fprintln(w, "...")
 	return err
 }
+
+//func initialize(fs afero.Fs, settings env.Settings, out io.Writer) error {
+func (i *initCmd) initialize() error {
```

Comment:
initialize on initCmd 🤔 I think this can be a plain function (it does not really belong to initcmd for me in any way - again we're not in OOP world). Also I would probably call this `initialiceLocalSystem`. What do you think?